### PR TITLE
Fix issue #685

### PR DESCRIPTION
**Pull request by adrianteri**
Fix/ Better document #685.

### CODE REVIEW

**Comment by adrianteri** [comment]
@LBGarber Added your changes on conciseness anything else?

File: website/docs/d/instances.html.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Each interface exports the following attributes:
 
 * `purpose` - (Required) The type of interface. (`public`, `vlan`)
 
-* `label` - (Optional) The name of this interface. If the interface is a VLAN, a label is required.
+* `label` - (Optional) The name of this interface. If the interface is a `vlan`, a label is required. Must be undefined for `public` purpose interfaces.
```
Comment (LBGarber) [inline website/docs/d/instances.html.md:175]: Just need to drop the trailing space here to get the docs linter passing
```suggestion
* `label` - (Optional) The name of this interface. If the interface is a `vlan`, a label is required. Must be undefined for `public` purpose interfaces.
```

**Review by LBGarber — CHANGES_REQUESTED** [review]
Looks great! Just one small fix to get the CI workflow passing

**Comment by adrianteri** [comment]
I think it's now all fixed. @LBGarber

**Review by LBGarber — APPROVED** [review]
Perfect! Thanks for the contribution :slightly_smiling_face:

**Pull request merged**